database/seeders/AccessibilitiesSeeder.php

```
@@ -0,0 +1,1489 @@
+                }
+            ]';
+        $data = json_decode($acccessibilites_json, true);
+        try {
+            DB::transaction(function () use ($data) {
+                collect($data)->map(function (array $row) {
+                    return Arr::only($row, ['sml_id', 'access_id', 'user_id']);
+                })->chunk(1000)
+                    ->each(function (Collection $chunk) {
+                        Accessibilities::upsert($chunk->toArray(), ['sml_id', 'access_id', 'user_id']);
+                    });
+            });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'run'. (Clean Code Rules)

(StaticAccess)